eth2util/deposit/deposit.go

```diff
@@ -105,6 +105,27 @@ func MarshalDepositData(pubkey eth2p0.BLSPubKey, withdrawalAddr string, network
 	return bytes, nil
 }
 
+// MarshalDepositDatas serializes a list of deposit data into a single file.
+func MarshalDepositDatas(depositDatas [][]byte) ([]byte, error) {
```

Comment:
is it fair to accept a list of serialized deposit datas and again deserialize the whole list here?

Reply:
removed this function in https://github.com/ObolNetwork/charon/pull/557